src/ssl.c

```diff
@@ -2829,6 +2829,60 @@ const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbyname(const char *name)
     return NULL;
 }
 
+const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbynid(int id)
+{
+    WOLFSSL_ENTER("EVP_get_cipherbynid");
+
+    switch(id) {
+
+#ifndef NO_AES
+        case NID_aes_128_cbc:
+            return wolfSSL_EVP_aes_128_cbc();
+        case NID_aes_192_cbc:
+            return wolfSSL_EVP_aes_192_cbc();
+        case NID_aes_256_cbc:
+            return wolfSSL_EVP_aes_256_cbc();
+#if defined(OPENSSL_EXTRA)
```

Comment:
Like this?

`#ifndef NO_AES
#if defined(OPENSSL_EXTRA)
        case NID_aes_128_cbc:
            return wolfSSL_EVP_aes_128_cbc();
        case NID_aes_192_cbc:
            return wolfSSL_EVP_aes_192_cbc();
        case NID_aes_256_cbc:
            return wolfSSL_EVP_aes_256_cbc();
        case NID_aes_128_ctr:
            return wolfSSL_EVP_aes_128_ctr();
        case NID_aes_192_ctr:
            return wolfSSL_EVP_aes_192_ctr();
        case NID_aes_256_ctr:
            return wolfSSL_EVP_aes_256_ctr();
        case NID_aes_128_ecb:
            return wolfSSL_EVP_aes_128_ecb();
        case NID_aes_192_ecb:
            return wolfSSL_EVP_aes_192_ecb();
        case NID_aes_256_ecb:
            return wolfSSL_EVP_aes_256_ecb();
#endif
#endif`